fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -362,6 +362,8 @@ class Settings(private val appContext: Context) : PreferencesHolder {
     val shouldShowSecurityPinWarning: Boolean
         get() = secureWarningCount.underMaxCount()
 
+    var shouldShowAccountDeletedSnackbar: Boolean = false
```

Comment:
Same comment as the others, we may not need this if we are observing state changes.

Reply:
same concern as above